specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2022-09-01/CertificateRegistrationProvider.json

```diff
@@ -0,0 +1,91 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-09-01",
+    "title": "CertificateRegistrationProvider API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.CertificateRegistration/operations": {
+      "get": {
+        "tags": [
+          "CertificateRegistrationProvider"
+        ],
+        "summary": "Implements Csm operations Api to exposes the list of available Csm Apis under the resource provider",
+        "description": "Description for Implements Csm operations Api to exposes the list of available Csm Apis under the resource provider",
+        "operationId": "CertificateRegistrationProvider_ListOperations",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2022-09-01/CommonDefinitions.json#/definitions/CsmOperationCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2022-09-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List operations": {
+            "$ref": "./examples/ListOperations.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {
+    "subscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "description": "Your Azure subscription ID. This is a GUID-formatted string (e.g. 00000000-0000-0000-0000-000000000000).",
+      "required": true,
+      "type": "string"
+    },
+    "apiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "description": "API Version",
+      "required": true,
+      "type": "string"
+    }
```

Comment:
Common parameters best practice - use common types
There are definitions of common parameters in commontypes types.json that you can use by $ref instead of declaring your own, like SubscriptionIdParameter, OperationIdParameter, and ApiVersionParameter,
See
[https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/common-types/resource-management/v3/types.json#L630](https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/common-types/resource-management/v3/types.json#L630)